community/vdr/vdr.post-deinstall

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+delgroup vdr video 2>/dev/null
+deluser vdr 2>/dev/null
```

Comment:
You should not remove users from the system in install scripts, you can only add them.